usaspending_api/awards/models_matviews/mv_search_award_all_pre2008.py

```diff
@@ -0,0 +1,88 @@
+from django.contrib.postgres.fields import ArrayField
+from django.contrib.postgres.search import SearchVectorField
+from django.db import models
+
+from usaspending_api.awards.models import Award
+
+
+class MatviewSearchAwardAllPre2008(models.Model):
```

Comment:
@tony-sappe For this and the other `SearchAward` matview models: I feel like there's some major opportunity here for inheritance. E.g. an Abstract Base Class, wherein the general structure of one of these matview awards models id defined. 

Since they can be unioned, and only differ by "type" of award, won't they all contain the same object attributes (and DB table columns)? 

That would make any future maintenance of these a lot simpler. Since there is potential for actually _re-combining_ these again if Postgres _partitioned_ tables can add value (after "tableview" refactor) ... having a single abstract base class with small derived models would again make that future work smaller.